README.md

```diff
@@ -131,7 +131,7 @@ git clone https://github.com/OSGeo/gdal.git
 cd gdal/gdal
 git checkout tags/2.1.0
 ./configure
-make install
+sudo make install
```

Comment:
I don't think so, but I must admit that I can't guarantee it. Feel free to skip this (or just add a comment).